7kyu/isograms.c

```diff
@@ -0,0 +1,27 @@
+#include <ctype.h>
+#include <stdbool.h>
+#include <stdio.h>
+#include <string.h>
+
+bool IsIsogram(char* str) {
+  int len = strlen(str);
+
+  for (int i = 0; i < len; i++) {
+    for (int j = i + 1; j < len; j++) {
+      if (toupper(str[i]) == toupper(str[j])) {
+        return false;
+      }
+    }
+  }
+  return true;
```

Comment:
Some people don't like multiple `return` statements in a function and prefer `break`s.